app/assets/stylesheets/catarse_bootstrap/_main.scss

```diff
@@ -4710,3 +4709,22 @@ html.w-mod-js *[data-ix="know-more-2"] {
     padding-left: 11px;
   }
 }
+
+@font-face {
```

Comment:
@thiagocatarse @diogob  we need to remove those @font-face rules from webflow's css.